pom.xml

```diff
@@ -367,22 +367,24 @@
             <version>${powsybl.core.version}</version>
         </dependency>
 
-        <!-- Test dependencies -->
+        <!-- Runtime dependencies -->
         <dependency>
             <groupId>com.powsybl</groupId>
-            <artifactId>powsybl-config-test</artifactId>
+            <artifactId>powsybl-iidm-impl</artifactId>
             <version>${powsybl.core.version}</version>
-            <scope>test</scope>
+            <scope>runtime</scope>
         </dependency>
+
+        <!-- Test dependencies -->
         <dependency>
             <groupId>com.powsybl</groupId>
-            <artifactId>powsybl-ucte-converter</artifactId>
+            <artifactId>powsybl-config-classic</artifactId>
```

Comment:
This is a bad idea to make test depend on the developer configuration config-test is better there.